### ISSUE TITLE
is user login a requirement that should be listed? 

### ISSUE THREAD

**Issue by PaulKlerkx** [MicrosoftDocs/windows-itpro-docs#11306]
Requirements - Does a user need to log in for bitlocker encryption to happen?

If yes, does the user account have specific requirements - E license, internet access etc

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3a4d34a9-22ce-1ad2-1590-e485a62ee08d
* Version Independent ID: 4aa3eea4-9311-d21a-35ee-ed8e9498b10e
* Content: [BitLocker overview and requirements FAQ (Windows 10)](https://learn.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-overview-and-requirements-faq)
* Content Source: [windows/security/information-protection/bitlocker/bitlocker-overview-and-requirements-faq.yml](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/information-protection/bitlocker/bitlocker-overview-and-requirements-faq.yml)
* Product: **windows-client**
* Technology: **itpro-security**
* GitHub Login: @frankroj
* Microsoft Alias: **frankroj**

**Comment by frankroj** [comment]
@PaulKlerkx No, there is no requirement for a user to log in for BitLocker Encryption to happen. BitLocker is an out of box Windows feature and does not require any additional user licenses. With that said, Pro, Enterprise, or Education versions of Windows are required for BitLocker.
